sdk/execution/sdk.go

```diff
@@ -14,19 +15,19 @@ import (
 
 // SDK is the execution sdk.
 type SDK struct {
-	client *cosmos.Client
-	kb     *cosmos.Keybase
+	client     *cosmos.Client
+	accAddress cosmostypes.AccAddress
 
 	serviceSDK  *servicesdk.SDK
 	instanceSDK *instancesdk.SDK
 	runnerSDK   *runnersdk.SDK
 }
 
 // New returns the execution sdk.
-func New(client *cosmos.Client, kb *cosmos.Keybase, serviceSDK *servicesdk.SDK, instanceSDK *instancesdk.SDK, runnerSDK *runnersdk.SDK) *SDK {
+func New(client *cosmos.Client, accAddress cosmostypes.AccAddress, serviceSDK *servicesdk.SDK, instanceSDK *instancesdk.SDK, runnerSDK *runnersdk.SDK) *SDK {
 	sdk := &SDK{
 		client:      client,
-		kb:          kb,
+		accAddress:  accAddress,
```

Comment:
We should get the `accAddress` from the `client` directly with something like `client.GetSignerAddress`